app/assets/javascripts/reviews.js

```diff
@@ -0,0 +1,28 @@
+$(document).ready(function(){
+  $('.review-answer').change(function(){
+    var el = $(this);
+    var reviewId = el.data('review-id');
+    var questionId = el.data('question-id');
+    var done = el.val();
```

Comment:
There seems to be a little problem with the ticking of the answers. Once the checkbox is ticked, the icon never goes back to red thumb down, even if you uncheck the checkbox. 

I think this might be because of the value `'yes'`, that never changes. Perhaps using `el.prop('checked')` here would solve the problem :)